protocols/v2/roles-logic-sv2/src/errors.rs

```diff
@@ -153,6 +154,7 @@ impl Display for Error {
             HashrateError(e) => write!(f, "Impossible to get Hashrate: {:?}", e),
             LogicErrorMessage(e) => write!(f, "Message is well formatted but can not be handled: {:?}", e),
             JDSMissingTransactions => write!(f, "JD server cannot propagate the block: missing transactions"),
+            MiningChannelError(e) => write!(f, "Mining channel error: {:?}", e),
```

Comment:
Could we come up with a more descriptive error message here?

```suggestion
            MiningChannel(e) => write!(f, "Mining channel error: {:?}", e),
```